d2renderers/d2svg/d2svg.go

```diff
@@ -1335,10 +1537,223 @@ func Render(diagram *d2target.Diagram, opts *RenderOpts) ([]byte, error) {
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.B6, lc, blendMode(lc))
+
+	// AA
+	lc, err = color.LuminanceCategory(theme.Colors.AA2)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.AA2, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.AA4)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.AA4, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.AA5)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.AA5, lc, blendMode(lc))
+
+	// AB
+	lc, err = color.LuminanceCategory(theme.Colors.AB4)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.AB4, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.AB5)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.AB5, lc, blendMode(lc))
+
+	// Neutrals
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N1)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N1, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N2)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N2, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N3)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N3, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N4)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N4, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N5)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N5, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N6)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N6, lc, blendMode(lc))
+	lc, err = color.LuminanceCategory(theme.Colors.Neutrals.N7)
+	if err != nil {
+		return "", err
+	}
+	out += fmt.Sprintf(".sketch-overlay-%s{fill:url(#streaks-%s);mix-blend-mode:%s}", color.N7, lc, blendMode(lc))
+
+	return out, nil
+}
+
+func blendMode(lc string) string {
+	switch lc {
+	case "bright":
+		return "darken"
+	case "normal":
+		return "color-burn"
+	case "dark":
+		return "overlay"
+	case "darker":
+		return "lighten"
+	}
+	panic("invalid luminance category")
```

Comment:
should never panic at runtime, just default normal

Reply:
well this should never happen... only if someone fricked up the code. will change it to a normal error and update the message to better explain it's not a user error

Reply:
Still reading through the project I think using panic here is reasonable. There is no way a user could change the category so it is basically unreachable code.